pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1613,6 +1626,9 @@
       |=  channel-id=@t
       ^-  [(list move) server-state]
       ::
+      ?~  connection-state=(~(get by connections.state) duct)
```

Comment:
This connection state checking shouldn't be needed with the new cancels, but this should fix the infinite behn loop state experienced by at least @loganallenc from my previous PR #1687.